cockpit/README.md

```diff
@@ -30,6 +42,11 @@ The most common commands are:
 
 (See https://yarnpkg.com for more details on using yarn).
 
+> When you are using our Nexus repository and you added some dependency using
+`yarn add <package>`, then please update `yarn.lock.npmjs` file. Vice verse, when
+you are community contributor and you added some package, then please update
+`yarn.lock.nexus` too.
+
```

Comment:
When using our Nexus repository and you add a dependency using
`yarn add <package>`, also update the `yarn.lock.npmjs` file. Conversely,
when you are community contributor and add a package, please update
`yarn.lock.nexus` as well.

Reply:
Thanks